src/envoy/mixer/config.cc

```diff
@@ -64,31 +59,54 @@ void ReadStringMap(const Json::Object& json, const std::string& name,
 }  // namespace
 
 void MixerConfig::Load(const Json::Object& json) {
-  ReadStringMap(json, kMixerAttributes, &mixer_attributes);
-  ReadStringMap(json, kForwardAttributes, &forward_attributes);
+  ReadStringMap(json, kMixerAttributes, http_config.mutable_mixer_attributes());
```

Comment:
I'm assuming the goal is for this PR to preserve existing behavior and v2 config will be added in a follow-up PR?

Reply:
Yes.  that is the plan.